frontend/src/App.svelte

```diff
@@ -29,6 +52,29 @@
 
     $: t = token
 
+
+    function getPatents() {
+        let contract = new $web3.eth.Contract(abi, '0xf8caa56A044b46f13655B2a07BbbCf1c94334981')
+        contract.methods.contract_id().call().then(
+            async el => {
+                console.log("el", el)
+                for (let i = 0; i <= el; i++) {
+                    await contract.methods.getPatent(i).call().then(
+                        c => {
+                            eth.patents.push(c);
```

Comment:
```suggestion
                            eth.update(e => {e.patents.push(c); return e});
```

See https://svelte.dev/docs#writable